worker/src/RTC/Codecs/VP8.cpp

```diff
@@ -325,7 +325,7 @@ namespace RTC
 			// clang-format off
 			if (
 				this->payloadDescriptor->hasTlIndex &&
-				this->payloadDescriptor->tlIndex > context->GetCurrentTemporalLayer()
+				this->payloadDescriptor->tlIndex == context->GetTargetTemporalLayer()
```

Comment:
Maybe we need to keep the packets with temporal layer < target ?

```suggestion
				this->payloadDescriptor->tlIndex <= context->GetTargetTemporalLayer()
```

Reply:
We must only **upgrade** the temporal layer with a packet whose temporal layer is the target one.